ca_description/urdf/create_apriltag.xacro

```diff
@@ -22,9 +20,9 @@
     <visual>
       <material>
         <script>
-          <uri>model://Camera_world/Apriltag3/materials/scripts/textures.material</uri>
-          <uri>model://Camera_world/Apriltag3/materials/textures/apriltag3.png</uri>
-          <name>Apriltag3</name>
+          <uri>model://camera_world/apriltag3/materials/scripts/textures.material</uri>
```

Comment:
I don't remember what I told you before, but `apriltag3` could be declared with uppercase (`AprilTag3`) because of naming convention. But It doesn't matter.